src/axom/spin/BVH.hpp

```diff
@@ -94,7 +94,7 @@ struct BVHPolicy<FloatType, NDIMS, ExecType, BVHType::LinearBVH>
  *     constexpr int DIMENSION = 3;
  *
  *     // get a list of axis-aligned bounding boxes in a flat array
- *     const double* aabbs = ...
+ *     const primal::BoundingBox<float, DIMENSION>* aabbs = ...
```

Comment:
Thanks @gunney1 -- this correction is certainly an improvement.

The situation is a bit more complicated since the code will work over any type that satisfies the `BoxIndexable` requirements. This includes `primal::BoundingBox` as well as `ZipIndexable<primal::BoundingBox>`.

We should probably document this better (in a future PR), e.g. here or in the BVH docs: https://axom.readthedocs.io/en/develop/axom/spin/docs/sphinx/bvh.html